pages/FilmFinder.js

```diff
@@ -28,23 +30,50 @@ const search = searchInput => {
     }
   })
 }
+
+const onClose = () => {
+  setSelectedFilm(null);
+}
+
+const renderSelectedFilm = () => {
+  if (selectedFilm) {
+        return (
+          <div className={styles.filmDetail} >
+            <FilmDetail className={styles.filmDetail} film={selectedFilm}/>
+            <button className={styles.closeButton}
+                width="50"
+                alt={"close button"}
+                onClick={onClose}>
+              close
+            </button>
+          </div>
+        )
+  }
+}
+
   return (
     <div>
       <div className={styles.fixedSearch}>
         <Header text="filmjack"/>
         <SearchButton search={search}/>
       </div>
-      <div className={styles.resultContainer}>
+      <div className={styles.resultWindow}>
         {loading && !errorMessage ? (
-          <span className={styles.loading}>loading...</span>
+          <span>loading...</span>
         ) : errorMessage ? (
           <div>{errorMessage}</div>
         ) : (
             films.map((film, index) => (
-              <Film key={index} film={film}/>
+              <div className={styles.filmCardContainers}>
+                <ul className={styles.listItem} key={index} onClick={() => {
+                  setSelectedFilm(film);
+                }}>
+                    <FilmCard film={film}/>
+                </ul>
+              </div>
             ))
```

Comment:
This html isn't sematically correct. Here is a good resources on HTML semantics https://learn-the-web.algonquindesign.ca/topics/html-semantics-cheat-sheet/ (The Lists section here should tell you what you need to know!)

Essentially you are rendering lots of lists with only one item here. Semantically you should wrap the entire thing in a `<ul>` and each film card should be a `<li>` item.